tasks/main.yml

```diff
@@ -55,6 +55,21 @@
   tags:
     - docker
 
+- name: Create networks
+  notify:
+    - Restart docker
```

Comment:
This doesn't require a daemon restart. Beside that, to create networks the daemon got to be running, so this must be moved to the bottom and maybe the handlers got to be flushed.